### PR TITLE
Update build.yml

### PR DESCRIPTION
**Pull request by prajin-op**
Update actions workflow

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -47,9 +46,13 @@ jobs:
         java-version: 11
     - name: Setup Apache Qpid Broker j
       run: |
-        wget https://downloads.apache.org/qpid/broker-j/9.1.0/apache-qpid-broker-j-9.1.0-src.tar.gz
-        tar -xvzf apache-qpid-broker-j-9.1.0-src.tar.gz
-        cd apache-qpid-broker-j-9.1.0-src/broker/bin
+        wget https://downloads.apache.org/qpid/broker-j/9.1.0/binaries/apache-qpid-broker-j-9.1.0-bin.tar.gz
+        tar -xvzf apache-qpid-broker-j-9.1.0-bin.tar.gz
+        cd qpid-broker/9.1.0/bin/
+        nohup ./qpid-server start &
```
Comment (navinkrishnan) [inline .github/workflows/build.yml:52]: There is additional start command at line 56. Is it needed ?

Comment (prajin-op) [reply .github/workflows/build.yml:52]: Yes, That is to start the Qpid server again after the config file update.

Comment (navinkrishnan) [reply .github/workflows/build.yml:52]: do we need to start before config file update as well ?

**Comment by prajin-op** [comment]
The config.json file will get generated after the Qpid-server start. And to change the Security mechanism we do update the file and do restart.

But interestingly I found that It works without restart of the Qpid-server. So I'm closing this PR and I have already updated the changes here https://github.com/SAP-samples/cloud-espm-cloud-native/pull/59.

**Pull request closed without merging**